src/user-event/__tests__/clear.test.tsx

```diff
@@ -0,0 +1,206 @@
+import * as React from 'react';
+import { View, TextInput, TextInputProps } from 'react-native';
+import { createEventLogger } from '../../test-utils/events';
+import { render } from '../../..';
+import { userEvent } from '../..';
+
+beforeEach(() => {
+  jest.useRealTimers();
+  jest.clearAllMocks();
```

Comment:
we already have `clearMocks: true` in the jest config so this line is not needed i think